packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -94,16 +94,10 @@ class TransactionService {
   }
 
   capturePageLoadMetrics(tr) {
-    var capturePageLoad = this._config.get('capturePageLoad')
-    if (
-      capturePageLoad &&
-      !this._alreadyCapturedPageLoad &&
-      tr.isHardNavigation
-    ) {
+    if (!this._alreadyCapturedPageLoad) {
       captureHardNavigation(tr)
       tr.addMarks(getPageLoadMarks())
-      self._alreadyCapturedPageLoad = true
-      return true
```

Comment:
Do we remember why we need this `this._alreadyCapturedPageLoad ` flag? 

In what scenario, the page load might be captured twice?

Reply:
I think `_alreadyCapturedPageLoad` was there to avoid a race conditions might be redundant at this point, that was also the reason for returning true if it was actually captured in this call!
Do you want to remove it in this PR?

Reply:
Yes, removed it now